src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -193,6 +206,27 @@ const DashboardEditor = ({
   const onDuplicateCard = (id) => duplicateCard(id);
   const onRemoveCard = (id) => removeCard(id);
 
+  const commonCardProps = (cardData, isSelected) => ({
+    key: cardData.id,
+    tooltip: cardData.description,
+    availableActions: { clone: true, delete: true },
+    onCardAction: (id, actionId) => {
+      if (actionId === CARD_ACTIONS.CLONE_CARD) {
+        onDuplicateCard(id);
+      }
+      if (actionId === CARD_ACTIONS.DELETE_CARD) {
+        onRemoveCard(id);
+      }
+    },
+    tabIndex: 0,
+    onKeyDown: (e) => handleKeyDown(e, onSelectCard, cardData.id),
+    onClick: () => handleOnClick(onSelectCard, cardData.id),
+    className: classnames(`${baseClassName}--preview__card`, {
+      // add black border when selected
+      [`${iotPrefix}--card--resizing`]: isSelected,
```

Comment:
I think you should use another class, since this one according to BEM has the modifier "resizing" to indicate it is being resized, which isn't correct in this case.

Reply:
Yep good point. I'll swap it to `--selected`

Reply:
Swapped to `__selected` which will fully be supported by the Card `isSelected` prop soon. Also added a TODO to add the `isSelected` prop to these common card props once that PR is ready